compass/step.py

```diff
@@ -340,6 +341,70 @@ def add_namelist_options(self, options, out_name=None, mode='forward'):
 
         namelist_list.append(dict(options=options, mode=mode))
 
+    def update_namelist_at_runtime(self, options, out_name=None):
+        """
+        Update an existing namelist file with additional options.  This would
+        typically be used for namelist options that are only known at runtime,
+        not during setup.  For example, the number of PIO tasks and the stride
```

Comment:
Should this sentence be removed now with the new function you've added specifically for PIO?

Reply:
Good suggestion.